app/models/label/LabelTable.scala

```diff
@@ -1170,19 +1171,18 @@ object LabelTable {
     */
   def selectLocationsAndSeveritiesOfLabels: List[LabelLocationWithSeverity] = db.withSession { implicit session =>
     val _labels = for {
-      (_labels, _labelTypes) <- labelsWithoutDeleted.innerJoin(labelTypes).on(_.labelTypeId === _.labelTypeId)
-    } yield (_labels.labelId, _labels.auditTaskId, _labels.gsvPanoramaId, _labelTypes.labelType, _labels.panoramaLat, _labels.panoramaLng)
```

Comment:
Thanks for noticing that we weren't even using `panoramaLat` and `panoramaLng`!  This query was written 4 years ago, and from the beginning they were never used :upside_down_face: